zebra-consensus/Cargo.toml

```diff
@@ -15,9 +15,12 @@ bellman = "0.10.0"
 bls12_381 = "0.5.0"
 chrono = "0.4.19"
 displaydoc = "0.2.2"
+halo2 = "=0.1.0-beta.1"
 jubjub = "0.7.0"
 lazy_static = "1.4.0"
 once_cell = "1.8"
+# TODO: replace with upstream orchard when these changes are merged
```

Comment:
Which changes?

Reply:
Ah I noted in the top level `Cargo.toml` not this one, resolved in 0689c695